.gitlab-ci.yml

```diff
@@ -230,7 +230,9 @@ SC_Service::package:
   ]
   script:
     - mkdir ./Service_Package/
+    - mkdir ./Service_Package/Licenses/
     - cp -r ./Build/* ./Service_Package/
+    - cp -r ./Third_Party_Licenses/* ./Service_Package/Licenses/
```

Comment:
We can build both in the same job i believe? Either way, this needs to be added yes, though not necessarily right now

Reply:
If we don't add the CI for this now, when will we? I'd rather add the CI for each new bit as we go now we have CI rather than having to do a catchup job for stuff we've forgotten how to run in a month or so